node/src/node/engine.cpp

```diff
@@ -571,7 +571,7 @@ auto engine_t::rebalance_slaves() -> void {
         target = *manual_target;
     } else {
         if (profile.queue_limit > 0) {
-            target = load / profile.grow_threshold;
+            target = (load + profile.grow_threshold - 1) / profile.grow_threshold;
```

Comment:
Can be simplified to `1 + (load-1)/profile.grow_threshold` or even with `std::ceil`.